### PR TITLE
Styling images

### LINKED ISSUE #32: Adjust images in stylesheet

### LINKED ISSUE #49: styling edit box for cart
The edit quantity form box for the cart is a bit enlarged.

### PR DESCRIPTION
**Pull request by CjMoore**
- formats image size on root
- formats image on cart
- standardizes item show card size
- adds number_to_currency helper in views

closes #32 
closes #49

### CODE REVIEW

File: app/views/shared/_items.html.erb [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
     <div class="card">
       <div class="card-image">
-        <img src="<%= item.image_tag %>" />
+        <img class="image-format" src="<%= item.image_tag %>" />
       </div>
       <div class="card-content">
          <span class="card-title"><%= link_to item.title, item_path(item) %></span>
          <h5>
-            <%= item.price %>
+            <%= number_to_currency(item.price, precision: 2) %>
```
Comment (meyerhoferc) [inline app/views/shared/_items.html.erb:11]: love this!

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -29,6 +29,13 @@
   #main {
     flex: 1 0 auto;
   }
+
+  .image-format {
```
Comment (meyerhoferc) [inline app/assets/stylesheets/application.css:33]: so relieved this is now uniform

**Review by meyerhoferc — APPROVED** [review]
looks great. This PR contains info from the other PR, so if we don't merge the one for orders / order-items, this PR will also need to be modified. @valerietrudell @cmacaulay

File: app/assets/stylesheets/application.css (hunk 2 above)
Comment (cmacaulay) [reply app/assets/stylesheets/application.css:33]: +1

**Review by cmacaulay — APPROVED** [review]
Looks good to me!

**Pull request merged**